testing/matrix-sdk-test/src/event_factory.rs

```diff
@@ -232,7 +232,12 @@ impl<E: StaticEventContent> EventBuilder<E> {
         self.state_key = Some(state_key.into());
         self
     }
+}
 
+impl<E: StaticEventContent> EventBuilder<E>
+where
+    E: Serialize,
+{
```

Comment:
This looks a bit weird. I'd move both bounds on `E` to the same place (probably the where bound).

Reply:
Changed.